example/passthrough_hp.cc

```diff
@@ -193,7 +195,14 @@ static void sfs_init(void *userdata, fuse_conn_info *conn) {
     if (conn->capable & FUSE_CAP_EXPORT_SUPPORT)
         conn->want |= FUSE_CAP_EXPORT_SUPPORT;
 
-    if (fs.timeout && conn->capable & FUSE_CAP_WRITEBACK_CACHE)
+    if (fs.passthrough && conn->capable & FUSE_CAP_PASSTHROUGH)
+        conn->want |= FUSE_CAP_PASSTHROUGH;
+    else
+        fs.passthrough = false;
```

Comment:
Hmm, no option for it? I personally use passthrough_hp to benchmarks for the kernel/userspace interface performance and also use it for xfstests - I think we need an option to at least disable passthrough.

Reply:
Ah sorry, ignore me, it gets late and I had missed the "fs.passthrough" condition.